src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Services/Implementation/PolicyAdministrationPoint.cs

```diff
@@ -231,27 +225,21 @@ private async Task<List<Rule>> ProcessPolicyFile(string policyPath, string org,
             string leaseId = await _policyRepository.TryAcquireBlobLease(policyPath);
             if (leaseId == null)
             {
-                _logger.LogInformation($"Could not acquire blob lease lock on delegation policy at path: {policyPath}", policyPath);
+                _logger.LogError("Could not acquire blob lease lock on delegation policy at path: {policyPath}", policyPath);
```

Comment:
Just curious if we know if using the params of the LogError method has any advatage over string interpolation? The alternative would have been:
```suggestion
                _logger.LogError($"Could not acquire blob lease lock on delegation policy at path: {policyPath}");
```

Reply:
I was told that adding them as parameters was giving better insight when analyzing as every parameter was logged seperatly as well as shown in the interpolated string. Using string interpolation and adding it again as parameter ended in an error in the actual log but worked fine in test so did not see the problem before it was run in swagger.